CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added links to source code in docs.
 - Fixed issue with GradientDescentTrainer when constructed with validation_data_loader==None and learning_rate_scheduler!=None.
+- Fixed issue with removing all handlers in root logger.
```

Comment:
This should be under the `### Fixed` section (you may need to create this section if it doesn't exist under the `## Unreleased` section.

Reply:
@epwalsh Does it look okay now? BTW, I've also moved the above fixed to this new section.